docs/Researcher/Walkthroughs/quickstart-inference.md

```diff
@@ -2,64 +2,196 @@
+* The project should be assigned a quota of at least 1 GPU. 
+* The URL of the Run:ai Console. E.g. [https://acme.run.ai](https://acme.run.ai).
+
+As described, the inference client can be created via CLI. To perform this, you will need to have the Run:ai CLI installed on your machine. There are two available CLI variants:
+
+* The older V1 CLI. See installation [here](../../admin/researcher-setup/cli-install.md)
+* A newer V2 CLI, supported with clusters of version 2.18 and up. See installation [here](../../admin/researcher-setup/new-cli-install.md)
 
 ## Step by Step Walkthrough
 
-### Setup
+### Login
+
+=== "CLI V1"
+    Run `runai login` and enter your credentials.
+
+=== "CLI V2"
+    Run `runai login` and enter your credentials.
+
+=== "User Interface"
+    Browse to the provided Run:ai user interface and log in with your credentials.
+
+=== "API"
+    To use the API, you will need to obtain a token. Please follow the [api authentication](../../developer/rest-auth.md) article.
+
+
+### Create an Inference Server Environment
+
+To complete this Quickstart __via the UI__, you will need to create a new Inference Server [Environment](../workloads/assets/environments.md) asset. 
+
+This is a __one-time__ step for all Inference workloads using the same image.
+
+Under `Environments` Select __NEW ENVIRONMENT__. Then select:
+
+* A default (cluster) scope.
+* Use the environment name `inference-server`.
+* The image `gcr.io/run-ai-demo/example-triton-server`.
+* Under `type of workload` select `inference`.
+* Under `endpoint` set the container port as `8000` which is the port that the triton server is using. 
+
 
-* Login to the Projects area of the Run:ai user interface.
-* Add a Project named "team-a".
-* Allocate 2 GPUs to the Project.
 
 ### Run an Inference Workload
 
-* In the Run:ai user interface go to `Inference`. If you do not see the `Inference` section you may not have the required access control, or the inference module is disabled.
-* Select `New workload` -> `Inference` on the top right.
-* Select `team-a` as a project and add an arbitrary name. Use the image `gcr.io/run-ai-demo/example-triton-server`.
-* Under `Resources` add 0.5 GPUs.
-* Under `Autoscaling` select a minimum of 1, a maximum of 2. Use the `concurrency` autoscaling threshold method. Add a threshold of 3.
-* Add a `Container port` of `8000`.
 
-This would start an inference workload for team-a with an allocation of a single GPU.
+=== "CLI V1"
+    Not available right now.
+
+=== "CLI V2"
+    Not available right now.
+
+=== "User Interface"
+    * In the Run:ai UI select __Workloads__
+    * Select __New Workload__ and then __Inference__
+    * You should already have `Cluster` and `Project` selected. Enter `inference-server-1` as the name and press __CONTINUE__.
+    * Under `Environment`,  select `inference-server`.
+    * Under `Compute Resource`, select `half-gpu`. 
+    * Under `Replica autoscaling, select a minimum of 0 and a maximum of 2. 
```

Comment:
you actually select a minimum of 1 and only when you select the scale to zero option later on, the minimum is automatically adjusted to 0.